index.html

```diff
@@ -155,11 +155,11 @@ <h3>Skills</h3>
 
         
        
-          <form action="">
+          <form action="https://formspree.io/f/xdojqgbe" method="post">
            
-           <input type="text" id="name" name="name" placeholder="Full name">
-            <input type="text" id="email" name="email" placeholder="Email Address">
-            <textarea name="comments" id="comments" cols="30" rows="10">Enter text here</textarea>
+           <input type="text" id="name" name="name" placeholder="Full name" minlength="30" required>
+            <input type="text" id="email" name="email" placeholder="Email Address" required>
+            <textarea name="comments"maxlength="500">Enter text here</textarea>
```

Comment:
- [ ] Kindy make sure to use the `placeholder` attribute in order to display the message for directing the user on what to write inside the particular text area. I highly recommend removing the static message and using the `placeholder` instead. :wink: